### DISCUSSION TITLE
Just a thought - a tutorial on how to make changes to add more features

### DISCUSSION THREAD

**Discussion by gganapat** [WLJSTeam/wljs-notebook#183] [General]
Since there are so many nooks and cranies in Mathematica that need to be filled, is it possible to make a tutorial on how you have been making the changes that people request (like me)? It will potentially speed up the fixes if more people pitch in. Right now, Mathematica (to me) is a big intimidating monolith, so I would not venture to even try making fixes to your code, but if the process was broken down into steps, perhaps I (or others) could jump in.

Thx for the great job you are doing!
gani -

**Comment by JerryI** [comment]
Thank you for your feedback :)

Yes, this is good idea. In principle it is easier than it looks and all changes I usually do during the development are like: change a few lines of code, reload the page, check the console, repeat.

I will come up with some tutorial on that and post it in docs.

**Comment by JerryI** [comment]
Finally there

https://wljs.io/frontend/Development/Overview
